### PR TITLE
[feat][Finder] Add command for sidebar icon size

### PR DESCRIPTION
**Pull request by jidicula**
**Additional context/notes/links**

Resolves yannbertrand/macos-defaults#66

### CODE REVIEW

**Comment by yannbertrand** [comment]
Thanks!

Would be great to add a screenshot on those. You should be able to copy from https://github.com/yannbertrand/macos-defaults/blob/master/record/finder/NSToolbarTitleViewRolloverDelay/0.5.js and take a screenshot instead of recording a video. Can you work on this as well?

**Comment by jidicula** [comment]
> Thanks!
> 
> Would be great to add a screenshot on those. You should be able to copy from https://github.com/yannbertrand/macos-defaults/blob/master/record/finder/NSToolbarTitleViewRolloverDelay/0.5.js and take a screenshot instead of recording a video. Can you work on this as well?

I'm a bit stuck on this. I tried following the [Dock orientation screencap config](https://github.com/yannbertrand/macos-defaults/blob/master/record/dock/orientation/bottom.js), but then I wasn't sure if I should instead be using `captureImage` since it allows a crop, rather than the entire screen, right?

**Comment by yannbertrand** [comment]
Yup `captureImage` would be easier to use and focus on what we want to capture, take a look at [this one](https://github.com/yannbertrand/macos-defaults/blob/2d8c73a4d054c245881ee95d2a52d9ca36581f28/record/dock/show-recents/false.js#L24) 🙂

**Comment by jidicula** [comment]
@yannbertrand I think it's ready for review, could you take a look? (sorry this took so long, I got swamped with other stuff this week 😅 )

**Review by yannbertrand — COMMENTED** [review]
Fixed a few issues and captured the whole Finder app 😄

**Comment by yannbertrand** [comment]
I'm not sure how to add these directly inside the PR, here are the screenshots:

[NSTableViewDefaultSizeMode.zip](https://github.com/yannbertrand/macos-defaults/files/5470491/NSTableViewDefaultSizeMode.zip)

**Comment by yannbertrand** [comment]
> sorry this took so long, I got swamped with other stuff this week 😅

No worries, Johanan. You've been great help on the project! Thank you for that 🙏

File: defaults.yml [hunk 1]
```diff
@@ -345,6 +345,22 @@ categories:
...
+          type: int
+        examples:
+          - value: 1
+            text: Small
+          - value: 2
+            default: true
+            text: Medium
+          - value: 3
+            text: Large
+        versions: [Catalina]
```
Comment (yannbertrand) [inline defaults.yml:362]:
```suggestion
        versions: [Big Sur, Catalina]
```

**Comment by jidicula** [comment]
> I'm not sure how to add these directly inside the PR, here are the screenshots:
> 
> [NSTableViewDefaultSizeMode.zip](https://github.com/yannbertrand/macos-defaults/files/5470491/NSTableViewDefaultSizeMode.zip)

Ok they're added 🚀

**Comment by yannbertrand** [comment]
Thank you again

**Pull request merged**